### DISCUSSION TITLE
How to operate if you want to measure the vertical and horizontal displacement of Moving particle mk=12

### DISCUSSION THREAD

**Discussion by duanlinlim** [DualSPHysics/DualSPHysics#255] [DUALSPHYSICS_V5.2]
In the CaseTurekHronFSI2 case, how to measure the vertical and horizontal changes in the simulated time of the elastic plate

**Comment by alexbexe** [comment]
One option is using PartVTK since you can get all data from one or a set of particles that you can save in different output formats.
